package.json

```diff
@@ -44,17 +44,20 @@
     "@patternfly/react-table": "6.1.0",
     "@patternfly/react-topology": "6.1.0",
     "@redhat-developer/vscode-redhat-telemetry": "^0.9.1",
+    "glob": "^11.0.0",
     "monaco-editor": "^0.50.0",
     "monaco-yaml": "5.2.3",
     "react": "18.3.1",
     "react-dom": "18.3.1",
-    "react-monaco-editor": "0.56.2"
+    "react-monaco-editor": "0.56.2",
+    "valid-filename": "^4.0.0",
+    "yaml": "^2.6.1"
   },
   "icon": "icon.png",
   "main": "./dist/extension/extension.js",
   "browser": "./dist/extension/extensionWeb.js",
   "engines": {
-    "vscode": "^1.74.0"
+    "vscode": "^1.95.0"
```

Comment:
needs to also edit this line in readme https://github.com/KaotoIO/vscode-kaoto/pull/779/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R8

Reply:
I don't see the connection to the engines parameter tbh.

Reply:
in the readme it is mentioning the minimal version of VS Code to use declaratively. This VS Code engine parameter is the one defining what is the minimal version the extension can be run with. it won't start on older versions.

Reply:
the engine update is needed because it is allowing import newer VS Code API